config/v1/metric_descriptor.proto

```diff
@@ -0,0 +1,171 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package config.v1;
+
+import "label_descriptor.proto";
+
+// Defines a metric type and its schema.
+//
+// A given metric is described by a set of labels. These labels represent the
+// different dimensions to associate with the metric. A given metric holds a
+// unique value for potentially any combination of these labels.
+//
+// Metric values can be integers, floating point numbers, strings, or booleans.
+//
+// Here's an example metric descriptor defined in a YAML configuration
+// file:
+//
+//    metrics:
```

Comment:
This looks good. 

But I prefer only mixer needs to know about this, no proxy.  The proxy just collects the facts (such as size, duration, ip) and send to mixer.